src/content/staking/pools/index.md

```diff
@@ -50,7 +50,7 @@ Attribute indicators are used below to signal notable strengths or weaknesses a
 There are a variety of options available to help you with your setup. Use the above indicators to help guide you through the tools below.
 
 <InfoBanner emoji="⚠️" isWarning>
-Please note the importance of choosing a service that takes <a href="/developers/docs/nodes-and-clients/client-diversity/">client diversity</a> seriously, as it improves the security of the network, and limits your risk. Services that have evidence of limiting majority client use are marked as <em style="text-transform: uppercase;">"diverse clients."</em>
+Please note the importance of choosing a service that takes <a href="/developers/docs/nodes-and-clients/client-diversity/">client diversity</a> seriously, as it improves the security of the network, and limits your risk. Services that have evidence of limiting majority client use are indicated with <em style="text-transform: uppercase;">"exeuction client diversity"</em> and <em style="text-transform: uppercase;">"consensus client diversity."</em>
```

Comment:
```suggestion
Please note the importance of choosing a service that takes <a href="/developers/docs/nodes-and-clients/client-diversity/">client diversity</a> seriously, as it improves the security of the network, and limits your risk. Services that have evidence of limiting majority client use are indicated with <em style="text-transform: uppercase;">" execution client diversity"</em> and <em style="text-transform: uppercase;">"consensus client diversity."</em>
```

Reply:
Thanks for catching... found it in one other place as well, just pushed a patch